### DISCUSSION TITLE
It's error with my deploy in vercel.com

### DISCUSSION THREAD

**Discussion by yph0** [rainbow-me/rainbowkit#2064] [Feedback]
The error problem is in the image below

How can I fixed it?
<img width="1440" alt="Screenshot 2567-06-25 at 01 48 05" src="https://github.com/rainbow-me/rainbowkit/assets/103312077/ee6b9d2f-e2f7-44f2-9227-69509eeda7c3">
![Untitled](https://github.com/rainbow-me/rainbowkit/assets/103312077/8983b20a-6106-4c00-9760-c4ca960c8071)

**Comment by 0xmkh** [comment]
@yph0 Can you try removing `package-lock.json` and `node_modules` and run `npm install` ? If that doesn't work do the same thing again, but instead of `npm install` do `npm install --legacy-peer-deps`.

We've had this same issue before. It's a peer dependency typescript error from Wagmi and Viem.

We also have a dApp example project which you can take a look if that's helpful https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next-app
